JASP-Engine/JASP/R/LSbinomialcommon.R

```diff
@@ -434,6 +449,51 @@
   return(support)
   
 }
+.modeBetaLS                 <- function(alpha, beta){
+  if(alpha == 1 && beta == 1){
```

Comment:
Those braces aren't necessary now are they?

Reply:
no, changed in this and many other cases